massgov/mayflower
# DP-20090 menu search align

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description
We were aligning these with absolute positioning. I switched this over to use flex, which should actually center these things more consistently across devices. I tested on iPhone 11 and Galaxy S10.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-20090)

## Steps to Test


1. Go to both the homepage and internal pages and check the alignment of the menu and search buttons along with their text

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
